File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/HTML_Content_Scaling_Sample_Code.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;

using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF
{
    public partial class HTML_Content_Scaling_Sample_Code : UserControl
    {
        public HTML_Content_Scaling_Sample_Code()
        {
            InitializeComponent();
        }

        private void HTML_Content_Scaling_Sample_Load(object sender, EventArgs e)
        {
            codeSampleWebBrowser.Visible = true;
            string codeSampleString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Code_Samples\CSharp\WinForms\HTML_to_PDF\HTML_Content_Scaling.html");
            codeSampleWebBrowser.DocumentText = codeSampleString;
        }
    }
}

File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/HTML_Content_Scaling_Sample_Code.vb
Imports System.ComponentModel

Imports System.Text

Namespace EvoHtmlToPdfDemo.HTML_to_PDF
	Partial Public Class HTML_Content_Scaling_Sample_Code
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub HTML_Content_Scaling_Sample_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			codeSampleWebBrowser.Visible = True
			Dim codeSampleString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Code_Samples\VbNet\WinForms\HTML_to_PDF\HTML_Content_Scaling.html")
			codeSampleWebBrowser.DocumentText = codeSampleString
		End Sub
	End Class
End Namespace
